iwyu.cc

```diff
@@ -1601,12 +1601,10 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     // the namespace of the decl.  If we have 'using std::vector;' +
     // 'std::vector<int> foo;' we don't actually care about the
     // using-decl.
-    // TODO(csilvers): maybe just insert our own using declaration
-    // instead?  We can call it "Use what you use". :-)
```

Comment:
Sentimental and silly, but this was one of my favorite comments, it always makes me smile :-)

At first I thought you removed it because you'd fixed the TODO, but it doesn't seem like it. so care to leave it to cheer me up?